src/Microsoft.Extensions.Hosting.Abstractions/IHostedService.cs

```diff
@@ -1,22 +1,20 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-namespace Microsoft.AspNetCore.Hosting
+namespace Microsoft.Extensions.Hosting
 {
     /// <summary>
     /// Defines methods for objects that are managed by the host.
     /// </summary>
     public interface IHostedService
     {
         /// <summary>
-        /// Triggered when the application host has fully started and the server is waiting
-        /// for requests.
+        /// Triggered when the application host has fully started.
         /// </summary>
         void Start();
```

Comment:
David, thx for the quick response! Would have been great to see this for 2.0! 

Won't your uber-host "completely break the world" by forcing a completely new pattern or did I understand it wrong?
```csharp
new HostBuilder()
  .UseLoggerFactory(...)
  .UseWebApplication(options => {
    // Would everyone currently targeting IWebHostBuilder have to target 'WebApplication' now?
    options.UseKestrel();
  })
```
Is your goal to allow people to register multiple services like the following?
```csharp
new HostBuilder()
  .UseLoggerFactory()
  .UseWebApplication()
  .UseABackgroundService()
  .UseAnotherBackgroundService();
```
I guess most applications will only host one service so this sounds like a very big breaking change for a very little used use case.

The common `IHostBuilder` approach sounds "less dramatic" and people who need multiple services could still do it by creating multiple hosts. The duplicated code wouldn't be too bad because the host-independent stuff should mostly be extension methods anyway. (e.g. UseSerilog(), UseStructureMap(), ...).